Orange/tests/test_txt_reader.py

```diff
@@ -48,29 +37,25 @@ def read_easy(self, s, name):
         try:
             file.write(s)
             file.close()
-            table = TxtFormat().read_file(filename)
+            table = CSVFormat().read_file(filename)
 
             f1, f2, f3 = table.domain.variables
-            self.assertIsInstance(f1, ContinuousVariable)
-            self.assertEqual(f1.name, name + "1")
+            self.assertIsInstance(f1, DiscreteVariable)
```

Comment:
All features are written as *.0 in the input file, aren't they supposed to be continuous?

Was the rule from
https://github.com/kernc/orange3/commit/f6e5ad69918427d78ebe20cc8deb8b224e6dd55a#diff-8fc32ad6a8ac706482fe82637339979aR46
droped?